cmake/DetectOptions.cmake

```diff
@@ -49,6 +49,16 @@ if(MGARD_FOUND)
   set(ADIOS2_HAVE_MGARD TRUE)
 endif()
 
+# PNG
+if(ADIOS2_USE_PNG STREQUAL AUTO)
+  find_package(PNG 1.6.0)
```

Comment:
Just one last comment: This broke on VTK-m's CI server; I downgraded it to PNG 1.2.0 and it was ok. (I dunno if you're using any PNG 1.6 features not present in PNG 1.2.)

Reply:
@NAThompson good call, I think I am....1.6 is pretty old by now and 1.2 ancient

Reply:
@williamfgc RHEL7 uses 1.5 and RHEL6 uses 1.2.  If there's not a hard requirement on v1.6 then we should be able to relax it some and just drop the version from the `find_package` call.

Reply:
@chuckatkins let me double check how much work would it be to downgrade. I am using the v1.6 high-level API for decompression following [their advice on releases](https://libpng.sourceforge.io/index.html) :

```
UPDATE 2019-Apr-14:

    The latest released version is
        libpng-1.6.37 

    [DOWNLOAD from SourceForge]

    For legacy applications,
        libpng-1.5.30
        libpng-1.4.22
        libpng-1.2.59
        libpng-1.0.69 
    are also available, and will be updated only if necessary for security reasons. 
```

Reply:
If you are using 1.6 specifically then that's fine.  The only caveat to realize is is that with RHEL7 you can't use the system libpng since it's only 1.5